ui/src/wizard/AzureWizard.ts

```diff
@@ -32,51 +31,49 @@ export class AzureWizard<T> implements types.AzureWizard<T>, IInternalAzureWizar
     }
 
     public get currentStep(): number {
-        return this._finishedPromptSteps.length + 1;
+        return this._finishedPromptSteps.filter(s => s.prompted).length + 1;
     }
 
     public get totalSteps(): number {
-        return this._finishedPromptSteps.length + this._promptSteps.filter(s => s.shouldPrompt(this._wizardContext)).length + 1;
+        return this._finishedPromptSteps.filter(s => s.prompted).length + this._promptSteps.filter(s => s.shouldPrompt(this._wizardContext)).length + 1;
```

Comment:
Why the +1?

Reply:
+1 is for the current step. Aka the step that has already been popped off of `this._promptSteps`, but won't be pushed to `this._finishedPromptSteps` until after the UI is shown

Reply:
Ah.